### PR TITLE
Record Collection - variable names shouldn't be keywords

### PR DESCRIPTION
**Pull request by jeremylt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes: N/A

I'd be happy with any other variable names, but we really shouldn't show learners naming a variable after a keyword.

Using a proper variable name has the side benefit of fixing the syntax highlighting for this problem.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md [hunk 1]
```diff
@@ -8,21 +8,21 @@ dashedName: record-collection
 
 # --description--
 
-You are given a JSON object representing a part of your musical album collection. Each album has a unique id number as its key and several other properties. Not all albums have complete information.
+You are given an object representing a part of your musical album collection. Each album has a unique id number as its key and several other properties. Not all albums have complete information.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md:11]:
```suggestion
You are given an object literal representing a part of your musical album collection. Each album has a unique id number as its key and several other properties. Not all albums have complete information.
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md [hunk 2]
```diff
@@ -8,21 +8,21 @@ dashedName: record-collection
...
 
-You start with an `updateRecords` function that takes an object like `collection`, an `id`, a `prop` (like `artist` or `tracks`), and a `value`. Complete the function using the rules below to modify the object passed to the function.
+You start with an `updateRecords` function that takes an object containing `records`, an `id`, a `prop` (like `artist` or `tracks`), and a `value`. Complete the function using the rules below to modify the object passed to the function.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md:13]:
```suggestion
You start with an `updateRecords` function that takes an object literal, `records`, containing the musical album collection, an `id`, a `prop` (like `artist` or `tracks`), and a `value`. Complete the function using the rules below to modify the object passed to the function.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
I like this change, and have always been irked by the use of a reserved word.

The first suggestion is me being pinnikity. The second is because, currently, it sounds as though there is a property key `records`.

**Comment by jeremylt** [comment]
That rewording makes sense to me.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 . Thanks, Jeremy.

**Review by Sembauke — APPROVED** [review]
LGTM 👍 🥇

**Pull request merged**